uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Service/PublicHolidayLeaveRequestDeletion.php

```diff
@@ -32,6 +32,26 @@ public function deleteForContact($contactID, PublicHoliday $publicHoliday) {
   }
 
   /**
+   * Deletes all the Public Holiday Leave Requests for Public Holidays in the
+   * future
+   */
+  public function deleteAllInTheFuture() {
+    $futurePublicHolidays = PublicHoliday::getAllInFuture();
+    $mostInTheFuturePublicHoliday = end($futurePublicHolidays);
```

Comment:
Do you think so? If you take a look at this page https://en.oxforddictionaries.com/definition/late, you can see that latest, most of the time, is used to mean something like: "most recent". Examples:

_"The latest news"_
_"Residents say this is only the latest in a whole string of accidents at the same spot."_

How about `$lastPublicHoliday`?